frontend/pages/hosts/details/OSSettingsModal/OSSettingsTable/OSSettingsTableConfig.tsx

```diff
@@ -53,6 +54,49 @@ interface IDataColumn {
   sortType?: string;
 }
 
+/**
+ * generates the formatted tooltip for the error column.
+ * the expected format of the error string is:
+ * "key1: value1, key2: value2, key3: value3"
+ */
+const generateFormattedTooltip = (detail: string) => {
+  const formattedText = detail.split(/[:,]+/).map((item, i, arr) => {
+    item.trim();
```

Comment:
```suggestion
    item = item.trim();
```


> The trim() method of [String](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String) values removes whitespace from both ends of this string and returns a new string, without modifying the original string.
> https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trim

Reply:
ah thanks! you know i remember looking this method up to see if it returned a new string, and I still got it wrong! 😆